### PR TITLE
Add package `reinput`

### PR DESCRIPTION
**Pull request by sospedra**
- [x] Added it to **react-native-libraries.json**

### CODE REVIEW

File: react-native-libraries.json [hunk 1]
```diff
@@ -5126,5 +5126,18 @@
...
+    "examples": ["https://github.com/sospedra/reinput/tree/master/example", "https://snack.expo.io/@git/github.com/sospedra/reinput:example"],
+    "images": ["https://user-images.githubusercontent.com/3116899/84693731-3d597380-af48-11ea-8614-b549211f5015.png"],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expo": true,
+    "windows": false,
+    "macos": false,
+    "unmaintained": false
+  },
```
Comment (Simek) [inline react-native-libraries.json:5142]: JSON validate failed because of trailing comma.

Comment (sospedra) [reply react-native-libraries.json]: ty, fixed :)

File: react-native-libraries.json [hunk 2]
```diff
@@ -5126,5 +5126,18 @@
     "expo": false,
     "windows": false,
     "macos": false
+  },
+  {
+    "githubUrl": "https://github.com/sospedra/reinput",
+    "npmPkg": "https://www.npmjs.com/package/reinput",
```
Comment (Simek) [inline react-native-libraries.json:5132]: Oh, I have just noticed that, sorry. You need to remove `npmPkg` or change it to the package name.

Comment (sospedra) [reply react-native-libraries.json]: np, done ^^

**Pull request merged**